cmd/cluster/core/create.go

```diff
@@ -164,14 +169,25 @@ func createCommonFixture(ctx context.Context, opts *CreateOptions) (*apifixtures
 		k, v := pair[0], pair[1]
 		annotations[k] = v
 	}
+	// Add annotation to support import into MCE
```

Comment:
why is this annotation being added unconditionally?
We have an --annotation flag in the cli that enable set annotations freely

Reply:
This helps with MCE import and general usability with MCE, but could just be documented as you suggest.

This annotation can be used to import the HostedCluster with a value other then that of the HostedCluster.Name.
For example, you could set this value to the infra-id when importing into MCE.

I'll remove it for now for simplicity.